src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public abstract class Figure implements Drawable {
+    public static final int NUMBER_OF_FIGURES = 5;
```

Comment:
let's move this constant to the FigureGenerator class)

Reply:
In my opinion, since this constant represents the total amount of classes, inherited from `Figure`, it would be better to store it here. If we want to change something in random generation logic, this constant has to stay the same anyway

Reply:
we can make these fields private and create constructor, getters or setters